components/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  Box,
+  chakra,
+  Container,
+  Stack,
+  Text,
+  useColorModeValue,
+  VisuallyHidden,
+} from '@chakra-ui/react'
+import { FaTwitter, FaGithub } from 'react-icons/fa'
+import { ReactNode } from 'react'
+
+const SocialButton = ({
```

Comment:
I'd recommend moving this component into its own file so we maintain one component per file.